code/src/components/ThoughtForm.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+const ThoughtForm = ( {onFormSubmit, newThought, setNewThought }) => {
+  return (
+    <form className="happy-form" onSubmit={onFormSubmit}>
+      <label htmlFor="newThought">What's making you happy right now?
+      <textarea
+        className="newThought"
+        id="newThought"
+        type="text"
+        placeholder="Type your thoughts here..."
+        autoComplete="off"
```

Comment:
This was a new thing for me... the `autoComplete="off"` Nice add! Made me google it so now I've learned something 😅